.github/workflows/ca-clone-replicated-ds-test.yml

```diff
@@ -136,16 +136,15 @@ jobs:
 
       - name: Enable replication on primary DS
         run: |
-          docker exec primaryds dsconf \
-              -D "cn=Directory Manager" \
-              -w Secret.123 \
-              ldap://primaryds.example.com:3389 \
-              replication enable \
-              --suffix=dc=ca,dc=pki,dc=example,dc=com \
-              --role=supplier \
-              --replica-id=1 \
-              --bind-dn="cn=Replication Manager,cn=config" \
-              --bind-passwd=Secret.123
+          docker exec secondary pki-server ca-db-repl-enable \
```

Comment:
The result should be the same but I would prefer to enable replication on primary DS from the `primary` instance

Reply:
Yeah, I was originally trying to mimic how these commands are executed by `pkispawn` (i.e. always from the secondary server), but I think that scenario has been tested by the regular CA cloning test, so in this test we can run the commands from the primary server.